.editorconfig

```diff
@@ -0,0 +1,20 @@
+root = true
+
+[**]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+max_line_length = 80
+
+# Set default charset and indentation for python files
+[{*.py, coalib/**/*.py, ./coala, tests/**/*.py}]
```

Comment:
@newbazz take a look at the samples on http://editorconfig.org/
And remove spaces after `,` and sorry for misguiding you earlier😅